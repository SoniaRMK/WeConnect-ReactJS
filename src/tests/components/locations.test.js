import {shallow} from 'enzyme';
import React from 'react';
import Locations from '../../components/business/locations';

describe('Locations component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<Locations/>)
        expect(wrapper).toMatchSnapshot();
    })
})
