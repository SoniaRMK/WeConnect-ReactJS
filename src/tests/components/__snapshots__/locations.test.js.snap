// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations component should match snapshot 1`] = `
<select
  className="form-control mb-2 mr-sm-2"
  name="location"
>
  <option
    value=""
  >
    Business Location
  </option>
  <option
    value="Mbarara"
  >
    Mbarara
  </option>
  <option
    value="Kampala"
  >
    Kampala
  </option>
  <option
    value="Gulu"
  >
    Gulu
  </option>
  <option
    value="Soroti"
  >
    Soroti
  </option>
  <option
    value="Hoima"
  >
    Hoima
  </option>
  <option
    value="Kabarole"
  >
    Kabarole
  </option>
</select>
`;
